x-pack/plugins/apm/public/style/variables.js

```diff
@@ -66,6 +68,17 @@ export const colors = {
   linkHover: colorBlue1
 };
 
+/**
+ * Takes a color string and returns its corresponding rgba string value
+ *
+ * @param {string} color - a polished compatible hex, rgb, rgba, hsl, or hsla color string
+ * @param {number} a - number between 0 and 1 to set the alpha value
+ */
+export function rgba(color, a = 1) {
+  const { red, green, blue, alpha = a } = parseToRgb(color);
+  return `rgba(${red},${green},${blue},${alpha})`;
+}
```

Comment:
Polished already has an `rgba` method: https://polished.js.org/docs/#rgba. Can't we use that?

```js
const { rgba } = require('polished');
const apmRed = '#920000';
console.log(rgba(apmRed, 0.5)) // rgba(146,0,0,0.5)
```

Reply:
lol I tried to use it and it blew up, but I now see it was because I omitted the alpha value.

Reply:
I'll swap that back out :)